src/components/SocialMediaLinks/SocialMediaLinks.tsx

```diff
@@ -0,0 +1,169 @@
+    /** ID that other components can cross reference for accessibility purposes */
+    id?: string;
+}
+
+const socialMediaMapData = [
+    {
+        type: 'facebook',
+        iconName: 'socialFacebook',
+        platformName: 'Facebook',
+        url: 'facebook.com'
+    },
+    {
+        type: 'twitter',
+        iconName: 'socialTwitter',
+        platformName: 'Twitter',
+        url: 'twitter.com'
+    },
+    {
+        type: 'instagram',
+        iconName: 'socialInstagram',
+        platformName: 'Instagram',
+        url: 'instagram.com'
+    },
+];
+
+function getLinksData(platforms) {
+    let allData = [];
+    platforms.forEach((myPlatform) => {
+        let thisPlatformName = (typeof myPlatform === 'string') ? myPlatform : myPlatform.type;
+        if (!socialMediaLinkTypeArray.includes(thisPlatformName)) {
+            throw new Error(
+                "NYPL Reservoir SocialMediaLinks: " + thisPlatformName + " is either not a supported platform or is misspelled.\n" +
+                "Supported platforms include: " + socialMediaLinkTypeArray.join(", ")
+            );
+        }
+        let thisPlatformArray = socialMediaMapData.filter( socialMediaSlice =>
+            socialMediaSlice.type === thisPlatformName);
+
+        // filter function returns an array of objects, so let's just get the objects out of the array.
+        let thisPlatformData = thisPlatformArray[0];
+
+        // If a url value exists in myPlatform (i.e. not undefined), use it instead of the official url.
+        let newUrl =  (typeof myPlatform.url !== "undefined") ? myPlatform.url : thisPlatformData.url;
+
+        let thisObj = {
+            type: thisPlatformData.type,
+            iconName: thisPlatformData.iconName,
+            platformName: thisPlatformData.platformName,
+            url: newUrl
+        };
+        allData.push(thisObj);
+    }) // end foreach
+
+    return allData;
+}
+
+/**
+ * The `SocialMediaLinks` component is a navigation element that provides a
+ * socialMediaLink path that reflects the site structure and allows a user to
+ * navigate to any page available in the socialMediaLink hierarchy.
+ */
+export const SocialMediaLinks = chakra(
+    forwardRef<HTMLDivElement, SocialMediaLinksProps>((props) => {
+        const {
+//            borders = "none",
+//            color = "textDefault",
+//            className,
+//            id,
+            linksData,
+            layout = "row",
+//            showLabels = false,
+//            size = "small",
+//            ...rest
+        } = props;
+
+
+        // If the linksData prop has data, set the map data to only include the passed values, else set it to
+        // include every platform.
+        const socialMediaDataArray  = (linksData) ? getLinksData(linksData) : socialMediaMapData;
+
+        const socialMediaLinkList= socialMediaDataArray.map(platformData => (
+                <Link href={platformData.url} rel="nofollow noopener noreferrer" key={platformData.type}>
```

Comment:
Yes, adding the `rel` is great! This actually happens automatically in the `Link` component. The link will always be external so you can do  `<Link href={platformData.url}  key={platformData.type} type="external">`

Internally, this will automatically add the rel value: https://github.com/NYPL/nypl-design-system/blob/development/src/components/Link/Link.tsx#L171

Reply:
Adding `type="external"` returns a visual indicator that is not in the design spec.

Reply:
Good point. @bigfishdesign13 any recommendations? Maybe the direct and manual `rel` and `target` props will be better in this situation.

Reply:
Yes, those attributes will need to be added manually.  As Andrew noted, using `type="external"` adds the "external" icon, so it is not an option.